pkg/kube/config.go

```diff
@@ -20,6 +20,7 @@ import (
 	"k8s.io/client-go/kubernetes"
```

Comment:
this file may belong in a different location.  I'm not quite sure.

Reply:
If the linter doesn't complain, why worry ? Most IDEs auto-order imports and collapse them.